inc/TRestHitsEvent.h

```diff
@@ -0,0 +1,90 @@
+
+#ifndef TRestSoft_TRestHitsEvent
+#define TRestSoft_TRestHitsEvent
+
+#include <TGraph.h>
+
+#include <iostream>
+
+#include "TArrayI.h"
+#include "TAxis.h"
+#include "TGraph2D.h"
+#include "TH2F.h"
+#include "TMath.h"
+#include "TObject.h"
+#include "TRestEvent.h"
+#include "TRestHits.h"
+#include "TVector3.h"
+
+//! An event data type that register a vector of TRestHits,
+//! allowing us to save a 3-coordinate position and energy.
+class TRestHitsEvent : public TRestEvent {
```

Comment:
I think that `TRestHitsEvent` should inherit from `TRestLegacyEvent` with some protection added